up42/cli.py

```diff
@@ -436,19 +436,30 @@ def add_workflow_tasks(workflow, input_tasks_json):
 
 @COMMAND_WORKFLOW
 @click.argument("input-parameters-json", type=click.File("rb"))
-@click.option(
-    "--test-query",
-    help="Only search for available imagery based on your data parameters",
-    is_flag=True,
-)
 @click.option("--track", help="Track status of job in shell.", is_flag=True)
 @click.pass_obj
-def create_and_run_job(workflow, input_parameters_json, test_query, track):
+def test_job(workflow, input_parameters_json, track):
+    """
+    Creates and runs a new job.
```

Comment:
Maybe add something here 
> Create a run a new test job  (Test Query). With this test query you will not be charged with any data or processing credits, but have a preview of the job result.

Reply:
Thanks, added also to other docstrings & docs examples!